org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/importing/CsarImporter.java

```diff
@@ -434,6 +434,9 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
                 this.adjustServiceTemplate(defsPath.getParent().getParent(), tmf, (ServiceTemplateId) wid, (TServiceTemplate) ci, errors);
                 entryServiceTemplate = Optional.of((ServiceTemplateId) wid);
             }
+            
+            // import license and readme files
```

Comment:
No need to add this comment.See [Java-by-Comparison](http://java.by-comparison.com/) issue "Remove Superfluous Comments"